README.md

```diff
@@ -0,0 +1,53 @@
+# sig-rpc.github.io
+
+The Reasonable Performance Computing SIG's Website & Knowledge-base, where user-submitted guides can help those writing software to identify and address performance problems in their code.
+
+The goal of the website is to make profiling and optimisation guidance accessible to even novice programmers. The main pre-requisite for readers is that they're able to read through code and recognise how it will execute.
+
```

Comment:
An actual link to the website could be useful in the readme? 

Could be:
- standalone
- just adding a hyperlink to the h1
- or as part of the first sentence (`[Website & knowledge base](https://sig-rpc.github.io)`)?

Not neccessary, just a suggestion from viewing the readme file in isolation (rather than from the repo root)